cmd/restic/global.go

```diff
@@ -84,6 +86,8 @@ func init() {
 	f.StringVar(&globalOptions.CacheDir, "cache-dir", "", "set the cache directory")
 	f.BoolVar(&globalOptions.NoCache, "no-cache", false, "do not use a local cache")
 	f.StringSliceVar(&globalOptions.CACerts, "cacert", nil, "path to load root certificates from (default: use system certificates)")
+	f.StringVar(&globalOptions.TLSClientCert, "tls-client-cert", "", "path to a TLS client certificate")
```

Comment:
Hm, can we have that in a single file? It looks like it's possible to use the `pem` decoding library to split a file into the key and the certificate(s). Let me know what you think and if you need help with that.

Reply:
I would even prefer a single file (cURL supports it, among others) however `tls.LoadX509KeyPair` definitely requires separate files, and reading a single file in and splitting it into certificate and key for use with `tls.X509KeyPair` is beyond my Go abilities, I'm sorry.